cypress/support/pages/course/pageCommonCourses.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+class Courses_Common {
+
+  static #learningContentTab = '[data-testid="learnContent"]'
+  static #toolsTab = '[data-testid="tools"]'
+  static #groupsTab = '[data-testid="groups"]'
+  static #toolsList = '[data-testid="course_tool_list_add_tool"]'
+  static #pageTitle = '[id="page-title"]'
+  static #courseOverviewNavigationButton = '[data-testid="Course-Overview"]'
+  static #addNewToolButton = '[data-testid="add_new_tool"]'
+
+  navigateToRoomsOverview() {
+    cy.get(Courses_Common.#courseOverviewNavigationButton).click()
+    cy.url().should('include', '/rooms-overview')
+  }
+
+  navigateToRoomBoard(room_name) {
+    const selectedRoom = `[aria-label='${room_name}']`
+    cy.get(selectedRoom).click({
+      multiple: true,
+      force: true
+    })
+  }
+
+  showRoomPage(room) {
+    const selectedRoom = `[aria-label='${room}']`
+    cy.get(selectedRoom).should('be.visible')
+  }
+
+  navigateToTools() {
+    cy.get(Courses_Common.#toolsTab).click()
+  }
+
+  addNewTool() {
+    cy.get(Courses_Common.#addNewToolButton).click()
+  }
+
+  courseIsVisibleOnOverviewPage(course_name) {
+    cy.url().should('include', '/rooms-overview')
+    cy.contains(course_name)
+  }
+
+  courseIsNotVisibleOnOverviewPage(course_name) {
+    cy.url().should('include', '/rooms-overview')
+    cy.contains(course_name).should('not.exist')
+  }
+
+  canAddBigBlueButton() {
```

Comment:
Here we are verifying that BBB is visible or not and not adding anything thing or doing any action i.e. toggle on/off. So name should be according to the method behaviour.